db/migrate/20230305005417_create_marketplace_tax_rates.rb

```diff
@@ -0,0 +1,14 @@
+class CreateMarketplaceTaxRates < ActiveRecord::Migration[7.0]
+  def change
+    create_table :marketplace_tax_rates, id: :uuid do |t|
+      t.float :tax_rate
```

Comment:
Float feels correct to me, since I agree that it could be a percent, or a fractional amount, or an integer fixed amount. (Maybe eventually we'll end up with an enum here to say if it is %age or "fixed".)